src/dev/mocha/run_mocha_cli.js

```diff
@@ -0,0 +1,75 @@
+import { resolve } from 'path';
+
+import getopts from 'getopts';
+import globby from 'globby';
+
+export function runMochaCli() {
+  const opts = getopts(process.argv.slice(2), {
+    alias: {
+      t: 'timeout',
+    },
+    boolean: [
+      'no-timeouts'
+    ],
+  });
+
+  const runInBand = process.execArgv.some(arg => {
+    switch (arg) {
+      case '--inspect':
+      case '--inspect-brk':
+        return true;
+    }
+  });
+
+  // ensure that mocha exits when test have completed
+  process.argv.push('--exit');
+
+  // ensure that mocha requires the babel-register script
+  process.argv.push('--require', require.resolve('../../babel-register'));
+
+  // set default test timeout
+  if (opts.timeout == null && !opts['no-timeouts']) {
+    if (runInBand) {
+      process.argv.push('--no-timeouts');
+    } else {
+      process.argv.push('--timeout', '10000');
+    }
+  }
+
+  // set default slow timeout
+  if (opts.slow == null) {
+    process.argv.push('--slow', '5000');
+  }
+
+  // set default reporter
+  if (opts.reporter == null) {
+    process.argv.push('--reporter', require.resolve('./server_junit_reporter'));
+  }
+
```

Comment:
__question:___ previously we had `ignoreLeaks: false`, should we include here something like `--check-leaks` by default?

Reply:
Huh, looks like you're right, double negative got me confused and I thought that did nothing.